### PR TITLE
Update types.md

### PR DESCRIPTION
**Pull request by chucklesb**
Fixed quotation syntax around `td` commands. Replaced references to `tl` command with `avga` command. Updated outputs of commands to reflect current behavior. Cleaned up some verbiage.

### CODE REVIEW

File: src/analysis/types.md [hunk 1]
```diff
@@ -239,19 +225,16 @@ COW
 
 ## Internal representation
```
Comment (chucklesb) [inline src/analysis/types.md:226]: Is any of the below information still valid?

Comment (chucklesb) [reply src/analysis/types.md:226]: Additionally, this document states that the typedefs are stored in Sdb, but I'm not sure if that's still the case.

**Review by XVilka — CHANGES_REQUESTED** [review]
.

File: src/analysis/types.md (hunk 1 above)
Comment (chucklesb) [reply src/analysis/types.md]: Ok, I have removed the "Internal Representation" section and mention of type definitions being stored in Sdb. Also small formatting and verbiage changes.

**Review by wargio — APPROVED** [review]
LGTM

**Review by XVilka — APPROVED** [review]

**Pull request merged**